File: Itan.Core/HomePageNews.cs
using System.Collections.Generic;

namespace Itan.Core
{
    public class HomePageNews
    {
        public List<LandingPageNews> TopNews { get; set; } = new List<LandingPageNews>();
        public List<LandingPageNews> BottomNews { get; set; } = new List<LandingPageNews>();
    }
}

File: Itan.Application.UI/src/app/components/landing-page/home-page.news.ts
import {HomePageNewsViewModel} from "../../../server/Itan/Core/HomePageNewsViewModel";
import {LandingPageNews} from "./landing-page.news";

export class HomePageNews {
    constructor(vm: HomePageNewsViewModel) {
        this.viewModel = vm;
        this.bottomNews = vm.bottomNews.map(n => new LandingPageNews(n));
        this.topNews = vm.topNews.map(n => new LandingPageNews(n));
    }

    private viewModel: HomePageNewsViewModel;
    topNews: LandingPageNews[];
    bottomNews: LandingPageNews[];
}
